pkg/gh/issues.go

```diff
@@ -13,17 +13,35 @@ type Issues struct {
 	RepoName    string
 	PackageName string
 	Comment     string
-	Retries     int
 	Title       string
 }
 
+func (o GitOptions) ListIssues(ctx context.Context, owner, repo, state string) ([]*github.Issue, error) {
+	var openIssues []*github.Issue
+
+	err := o.handleRateLimitList(func(opt *github.ListOptions) (*github.Response, error) {
+		ilo := github.IssueListByRepoOptions{
+			State:       state,
+			ListOptions: *opt,
+		}
+		issues, resp, err := o.GithubClient.Issues.ListByRepo(ctx, owner, repo, &ilo)
```

Comment:
You silently ignored the `err` here. [ListByRepo](https://github.com/google/go-github/blob/521fd0d1f06050e4fe370eb5eaa02a10d9e50baa/github/issues.go#L222-L244) returns `nil` on any error case. But I think this works since current code base already using in this way.

Reply:
the err is returned on line 29 unless I miss understand the comment here?

Reply:
going to close this but happy to followup if there's something missed.